doc/ideas/gameplay.md

```diff
@@ -179,25 +358,81 @@ New Buildings
  - Blocks ships, allows land units to walk
  - Forgotten Empires might already have wooden bridge textures
  - Destructible, repairable, ...
- - Long build time
+ - construction per tile
+ - "terraforming"-style
+   - build a construction into the sea and build docks & harbours at it
+ - Long build time per tile
+ - wooden bridge with less HP, stone bridge with more HP and faster walking speed?
+
+### Streets
+
+ - Ability to build streets on land
+   - textures already exists
+ - increases the speed of trade carts, vills, siege or in general land units
+ - destructible just by mangonel-line or trebuchet (ground attack units) and maybe decay over time or usage, repairable (or not?)
+ - uses stone, maybe 1/3 the cost of a stonewall
+
+ - Ideas of the original developers:
+   - Roads improve trade by allowing trade units to move more quickly over the terrain
+   - Neutral to all players
+   - Build automatically
+   - Progressive
+   - Decay when not used
+   - Appearance and quality depends on traffic (matted grass, dirt path, etc.)
+
+New Units
+-------------
 
+### Water
 
-Resources
+ - monk-ship to convert ships on water
+
+### Land
+
+ - transport waggon to slowly transport ships over land from one pond to another
+
+
+Relics & Kings
 ---------
 
-### Infinite regeneration
+ - Relics could have special abilities like in AOM
+   - e.g. they can have attack bonuses (patches) for special units or economic/military bonuses
```

Comment:
patches have another meaning in nyan. I think the "(patches)" part of the sentence could be left out.

Reply:
done